odfuzz/fuzzer.py

```diff
@@ -368,6 +368,22 @@ def generate_body(self,accessible_entity,key_pairs):
                 except:
                     pass
                 body[prprty._name] = generated_body
+        elif Config.fuzzer.http_method_enabled == "MERGE":
+            properties = deepcopy(accessible_entity.entity_set.entity_type._properties)
```

Comment:
What is the purpose of the deepcopy..? Why the PUT and POST are same and just generating stuff, while MERGE needs to copy the structure and than call some randomized stuff on it? Why the body content is dependent on random? What if we want to generate MERGE requests with specific properties, not select them on random?

Each if/elif block seems to me needs some kind of comments explaining the rationale behind. Consider the blocks to be separate functions (which they will probably will be refactored to in the future, with its unit tests) and think of pydocs for them. Functions would probably serve better if we would like to - and we will - to get rid of random.something() scattered trough the codebase and have the Requests generation better configurable, say to some TestStrategy.

Reply:
addressed as discussed